docs/releases/1.11.6.txt

```diff
@@ -11,3 +11,6 @@ Bugfixes
 
 * Made the ``CharField`` form field convert whitespace-only values to the
   ``empty_value`` when ``strip`` is enabled (:ticket:`28555`).
+
+* Ensured that CSRF processing works as expected when an error handler
```

Comment:
suggested wording: Fixed a regression in Django 1.9 where a custom view error handler such as ``handler404`` that accesses ``csrf_token`` (for example, by using ``django.shortcuts.render()``) could cause CSRF verification failures on other pages

Reply:
Using `render` shouldn't actually cause this issue since the token is used lazily.